### ISSUE TITLE
Clear confusion in build.classify() documentation.

### ISSUE THREAD

**Issue by raspberry-soft** [renpy/renpy#3548]
I used the `build.package()` function to build a patch this way and worked:

```
build.archive('patch_name', 'patch_file_list')
build.classify(pattern, 'patch_name')
build.package('patch', format='directory', file_lists='patch_file_list')
```

However, the doc doesn't give an example of the first 2 lines, just the 3rd, and it took me a lot of trial and error to get it right. The reason was confusion regarding this line in the doc:

`build.classify(pattern, file_list)`

But my first example and even the docs itself imply the function should be used like this:

`build.classify(pattern, archive name)`

```
# Declare two archives.
build.archive("scripts", "all")
build.archive("images", "all")

# Put script files into the scripts archive.
build.classify("game/**.rpy", "scripts")
build.classify("game/**.rpyc", "scripts")

# Put images into the images archive.
build.classify("game/**.jpg", "images")
build.classify("game/**.png", "images")
```

If I use the function as the doc explain the patch would not be built.

```
build.archive('patch_name', 'patch_file_list')   # Docs: build.archive(name, file_list=u'all')
build.classify(pattern, 'patch_file_list')   # Docs: build.classify(pattern, file_list)
build.package('patch', format='directory', file_lists='patch_file_list')
```

If it weren't for the examples of how to use `build.classify()` I would never get it just by reading the function doc. Also it should specify that you can create new file lists using that function.

**Comment by Gouvernathor** [comment]
To provide more clarity, the problem in the doc doesn't come from the explanations at the top of [the page](https://www.renpy.org/doc/html/build.html), which I think are not misleading (even if it could always be made clearer), but from the functions list and signatures at the [bottom of the page](https://www.renpy.org/doc/html/build.html#build-functions), where this happens :
![image](https://user-images.githubusercontent.com/44340603/169647200-e4844d61-a1df-4f21-92db-7a4d27f0c65c.png)
Basically the second argument of the second function should match the first argument of the first function, but it's shown to match the second argument of the first function instead.
As these are the hardcoded arguments names, they can't be modified in a whim between two versions, but I'll see how it could be made clearer.
